index.html

```diff
@@ -0,0 +1,74 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title> CLARENCE PORTFOLIO</title>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap" rel="stylesheet">
+</head>
+
+
+<body>
+  
+  <header class="header" id="header">
+    <a href="#">CLARENCE</a>
+    <nav>
+      <span class="open" id="menu"></span>
+       <img src="images/Icon - Menu.png" alt="menu bar" class="icon-container">
+      
+    </nav>
+  </header>
+  
+  <!---Main---->
+
+  <main class="main">
+      
+    
+          <div class="home-page">
+              <div>
+                  <h2 class="home_header">Hey There.</h2>
+                  <h3 class="home_header">I'm Clarence Atim</h3>
+              </div>
+
+              <div class="home-description">
+                  <h5 class="intro-heading">I'm a Software Developer</h5>
+
+                  <p class="subject">
+                      I can help you build a product, feature or website Look through some
+                      of my work and experiece! If you like what you see and have a
+                      project you need coded, don't hesitate to contact me.
+                  </p>
+              </div>
+          </div>
+
+          <!-- media icons-->
+            <ul class="icons">
+              <li>
+                  <a href="#main"><img src="./images/Icon -GitHub.png" class="social-media-icon" /></a>
+              </li>
+              <li>
+                  <a href="#main"><img src="./images/Icon - Linkedin.png" class="social-media-icon" /></a>
+              </li>
+              <li>
+                  <a href="#main"><img src="./images/Icon - Angelist.png" class="social-media-icon" /></a>
+              </li>
+              <li>
+                  <a href="#main"><img src="./images/Icon - Twitter.png" class="social-media-icon " /></a>
+              </li>
+              <li>
+                  <a href="#main"><img src="./images/Icon - Medium.png" class="social-media-icon" /></a>
+              </li>
+          </ul>
+
+          <!--illustrations-->
+           <img src="./images/illustration-header-mobile.png" alt="header-mobile" class="illustration-header">
```

Comment:
Because this image is only a design component and not necessarily part of the page's content, kindly use a background image instead. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.